crates/sui-framework/sources/types.move

```diff
@@ -7,5 +7,5 @@ module sui::types {
 
     /// Tests if the argument type is a one-time witness, that is a type with only one instantiation
     /// across the entire code base.
-    public native fun is_one_time_witness<T: drop>(_: T): bool;
+    public native fun is_one_time_witness<T: drop>(_: &T): bool;
```

Comment:
I'm still very nervous about this. Can we just make the `balance::create_supply` to be `&T`?

Reply:
But maybe we are being overly prescriptive. If you want your function to be one-time, take the witness owned. If you don't care, take it by ref, and you at least know its privileged (more so than normal)?

Reply:
While I'm warming to the by reference here, it changes the name of `one-time` to really being `one-instantiation`, where you can use it over and over (if the application allows you to)

Reply:
Though if we make `balance::create_supply` by reference. It causes some other headaches, so maybe not a good idea...

Reply:
Yeah, I wouldn’t go that route. Balance has to consume the witness.